packages/core/.babelrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
in babel7 (soon-ish) it will be possible to use `.babelrc.js` directly and therefore omit those `.babelrc` entirely